### PR TITLE
CephFS-Tier-2-CEPH-11298

### PR DESCRIPTION
**Pull request by hkadam134**
Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-4GUYLI/
Duration of Run : 00:07:25

Signed-off-by: Hemanth <hyelloji@redhat.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/clients/rsync_bw_fs_and_other_location.py [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+        stop = timeit.default_timer()
+        total_time = stop - start
+        mins, secs = divmod(total_time, 60)
+        hours, mins = divmod(mins, 60)
+        print("Hours:%d Minutes:%d Seconds:%f" % (hours, mins, secs))
+        return 0
+    except CommandFailed as e:
+        log.error(e)
+        log.error(traceback.format_exc())
+        log.info("Cleaning up!-----")
```
Comment (AmarnatReddy) [inline tests/cephfs/clients/rsync_bw_fs_and_other_location.py:221]: if possible can we move the cleanup part to finally block

File: tests/cephfs/clients/rsync_bw_fs_and_other_location.py [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+from ceph.ceph import CommandFailed
+from ceph.parallel import parallel
+from tests.cephfs.cephfs_utils import FsUtils
+from tests.cephfs.cephfs_utilsV1 import FsUtils as FsUtilsV1
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
```
Comment (AmarnatReddy) [inline tests/cephfs/clients/rsync_bw_fs_and_other_location.py:13]: Can we add doc string for this test

**Comment by hkadam134** [comment]
@AmarnatReddy Updated the changes as per the comments.

**Review by AmarnatReddy — APPROVED** [review]
LGTM

**Pull request merged**